NEWS.md

```diff
@@ -41,6 +41,8 @@
 ### Misc
 
 * **BREAKING:** RStudio Desktop Pro only supports activation with license files (Pro #2300)
+* **BREAKING:** The `session-env-var-save-blacklist` option has been renamed to `session-ephemeral-env-vars`.
```

Comment:
Do you think it's worth supporting the old options? (I'm guessing these are used somewhat rarely, so perhaps that's okay?)

Reply:
Yes, these the `save-blacklist` was introduced for one customer's use case AFAIK, and I added the `view-whitelist` as an escape hatch but to my knowledge no one has actually needed to use it.